src/Features/CSharp/Portable/EditAndContinue/BreakpointSpans.cs

```diff
@@ -770,6 +769,63 @@ private static TextSpan CreateSpanForBlock(BlockSyntax block, int position)
     private static SyntaxToken LastNotMissing(SyntaxToken token1, SyntaxToken token2)
         => token2.IsKind(SyntaxKind.None) || token2.IsMissing ? token1 : token2;
 
+    private static TextSpan? TryCreateSpanForLocalDeclarationStatement(LocalDeclarationStatementSyntax declarationStatement, int position)
+    {
+        var variableDeclaration = declarationStatement.Declaration;
+        if (variableDeclaration.Variables.Count == 0)
+        {
+            return null;
+        }
+
+        if (declarationStatement.Modifiers.Any(SyntaxKind.ConstKeyword))
+        {
+            // no sequence points are emitted for const locals
+            return default(TextSpan);
+        }
+
+        if (variableDeclaration.Variables.Count == 1)
+        {
+            if (variableDeclaration.Variables[0].Initializer == null)
+            {
+                return default(TextSpan);
+            }
+
+            // Include 'await' and 'using' keywords in the span if present
```

Comment:
i think in this case the start can just be full local-decl-statement itself.  then we dont need to special case await/using/modifiers/var-decl.